src/main/java/core/basesyntax/garage/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.garage;
+
+public class Bulldozer extends Machine{
```

Comment:
There should be a space between the class name and the curly brace to follow Java coding conventions.